arbitrum-docs/stylus/how-tos/adding-support-for-new-languages.mdx

```diff
@@ -0,0 +1,304 @@
+---
+title: 'Adding Stylus support for new programming languages'
+sidebar_label: 'Adding support for new programming languages'
```

Comment:
```suggestion
sidebar_label: 'Add a new programming language'
```

Reply:
weakly held suggestion -> removing precision in favor of tight sidebar title & page title; the intro of the body sets the stage more precisely